lopper/assists/openamp_xlnx.py

```diff
@@ -632,6 +632,14 @@ def xlnx_rpmsg_parse(tree, node, openamp_channel_info, verbose = 0 ):
     return True
 
 
+# tests for a bit that is set, going fro 31 -> 0 from MSB to LSB
+def check_bit_set(n, k):
+    if n & (1 << (k)):
+        return True
+
+    return False
+
+
```

Comment:
This is also available in lopper_lib, so you can import it from there. A potential cleanup idea for future changes.